config/locales/en.yml

```diff
@@ -294,10 +314,85 @@ en:
         default: Use as default outgoing mail address
         verification_token: Verified
 
+      email_template:
+        name: Template name
+        from: From
+        subject: Subject
+        kind: Kind
+        message: Message
+        kinds:
+          user_welcome:
+            template_name: Default user welcome
+            message: |
+              <p>
+              Dear [name],
+              <br />
+              </p>
+              <p>
+              Your account for the open source ticket system BRIMIR has just been created!
```

Comment:
Let's make it noob friendly: leave "open source" out and change BRIMIR to Brimir.